.pre-commit-config.yaml

```diff
@@ -35,9 +35,10 @@ repos:
         additional_dependencies:
           - "backoff>=2.2.1,<3"
           - "click>=8.1.7,<9"
+          - "jinja2>=3.1.2,<4"
```

Comment:
pre-commit gives me this error `site-packages\pandas-stubs\io\formats\style_render.pyi:11: error: Cannot find implementation or library stub for module named "jinja2"  [import-not-found]` otherwise

Reply:
alrighty, will be gone with MX-1515 anyway.